docs/pages/getting_started/access_data.mdx

```diff
@@ -0,0 +1,99 @@
+# Getting Started: Accessing Data
+The developers of the Pelican Platform believe data is the lifeblood of science. That's why Pelican provides a suite of tools that make it easy for researchers to federate their data, giving them the ability to share objects from a broad range of data repositories and allowing them to connect their data with distributed computational capacity, such as the OSPool. For more information on Pelican, visit the [About Pelican](../index.mdx) page.
+
+## Before Starting
+
+### Assumptions
+
+Before using the Pelican client to interact with objects, this guide makes several assumptions:
+
+- You are on a computer where you have access to a terminal. The Pelican client is a command line tool.
+- You've already installed the version of Pelican appropriate for your system, and Pelican is accessible via your path. To test this on Linux, you can run
+
+  ```bash
+  which pelican
+  ```
+  which should output a path to the executable. If there is no output to this command, refer to the [Pelican installation docs](../install.mdx) to acquire a working installation.
+
+### Useful Terminology
+
+#### Core Concepts
+To learn our terminology and the basics of Pelican, we recommend you read through our [Core Concepts](../index.mdx) page to learn how Pelican works and how it can be useful to you.
```

Comment:
I find it strange that we link to a page called "Core Concepts" when we're already in a section called "Core Concepts." Can we remove the `#### Core Concepts` header and simply have the sentence "To learn additional terminology and about Pelican's architecture, we recommend you read ..." under the `### Useful Terminology` header?

Reply:
Since the core concepts section is still under review, and I don't want to leave a broken link in the doc. Let's update this as a separate ticket as both are merged in. https://github.com/PelicanPlatform/pelican/issues/1396